src/tekhsi/tek_hsi_connect.py

```diff
@@ -489,11 +489,25 @@ def done_with_data(self) -> None:
         self._done_with_data_release_lock()
 
     def force_sequence(self) -> None:
-        """force_sequence asks the instrument to please give us access.
-
-        to the current acquisition data. This is useful when connecting to a stopped instrument to
+        """force_sequence asks the instrument to please give us access to the already acquired data. 
+        This is useful when connecting to a stopped instrument to
```

Comment:
There needs to be a blank line between the docstring summary and the rest of the details.
```suggestion

        This is useful when connecting to a stopped instrument to
```